labgrid/strategy/common.py

```diff
@@ -25,6 +26,10 @@ def __attrs_post_init__(self):
             raise BindingError(
                 "Strategies can only be created on a valid target"
             )
+        # Open the ConsoleProtocol to not race the output after reset
+        # FIXME: If we have more than one Driver that implements this protocol
+        # we need to iterate over all of them
+        self.target.activate(self.target.get_driver(ConsoleProtocol))
```

Comment:
We should instead do this in the early steps in a specific strategy. The bound drivers are also available there.